napari/__main__.py

```diff
@@ -187,5 +188,46 @@ def main():
             )
 
 
+def _run_pythonw():
+    """Execute this script again through pythonw.
+    This ensures we're using a framework build of Python on macOS.
+    """
+    import pathlib
+    import subprocess
+
+    cwd = pathlib.Path.cwd()
+    python_path = pathlib.Path(sys.exec_prefix) / 'bin' / 'pythonw'
+
+    if not python_path.exists():
+        msg = (
+            'pythonw executable not found. '
+            'Please install python.app via conda.'
```

Comment:
i wonder if we want to give the exact instructions: `conda install -c conda-forge python.app`
some people might not notice the ".app" and think that they already have it installed

Reply:
also... do we actually want to raise an exception and completely prevent napari from running here just because the menubar won't be active unless they refocus?  Maybe a warning instead?